tree-sitter-go/tree-sitter-go.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                tree-sitter-go
-version:             0.5.0.0
+version:             0.5.0.1
```

I think this is 0.5.1.0, since you’re adding functionality
```suggestion
version:             0.5.1.0
```

---

Since this doesn't change the runtime api (just makes some test data available), I'm going to leave it as a patch bump.